### PR TITLE
Np 595 update sar

### PR DESCRIPTION
**Pull request by svenngjeto**
Updated deployment metadata, removed cors parameters

### CODE REVIEW

File: template.yaml [hunk 1]
```diff
@@ -89,7 +76,7 @@ Resources:
     Properties:
       Environment:
         Variables:
-          AllowOrigin: !Ref  CorsOrigin
+          AllowOrigin: "'*'"
       Handler: no.unit.nva.cristin.institutions.FetchCristinUnit::handleRequest
       Runtime: java8
```
Comment (janbj) [inline template.yaml]: java11 runtime?

File: template.yaml [hunk 2]
```diff
@@ -64,7 +51,7 @@ Resources:
     Properties:
       Environment:
         Variables:
-          AllowOrigin: !Ref  CorsOrigin
+          AllowOrigin: "'*'"
```
Comment (janbj) [inline template.yaml]: Env variable names should be upper case

**Review by janbj — CHANGES_REQUESTED** [review]

File: template.yaml (hunk 1 above)
Comment (svenngjeto) [reply template.yaml]: Ups! Dette var sløvt...

File: template.yaml (hunk 2 above)
Comment (svenngjeto) [reply template.yaml]: Done!

File: template.yaml [hunk 3]
```diff
@@ -89,9 +76,9 @@ Resources:
     Properties:
       Environment:
         Variables:
-          AllowOrigin: !Ref  CorsOrigin
+          ALLOWORIGIN: "'*'"
```
Comment (janbj) [inline template.yaml]: I would have prefered ALLOW_ORIGIN :)

File: template.yaml [hunk 4]
```diff
@@ -89,9 +76,9 @@ Resources:
...
       Handler: no.unit.nva.cristin.institutions.FetchCristinUnit::handleRequest
-      Runtime: java8
+      Runtime: java11
       MemorySize: 512
       Events:
         NvaCristinFetchUnitEvent:
```
Comment (janbj) [inline template.yaml:84]: I think you are missing a RequestParameter for {id}

**Review by janbj — CHANGES_REQUESTED** [review]

File: template.yaml (hunk 4 above)
Comment (svenngjeto) [reply template.yaml:84]: Added

File: template.yaml (hunk 3 above)
Comment (svenngjeto) [reply template.yaml]: Jada, skla få det

**Review by janbj — APPROVED** [review]

**Pull request merged**